Sentinel/Classes/EmailSender/EmailSenderTool.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  EmailSenderTool.swift
+//  Sentinel
+//
+//  Created by Antonijo Bezmalinovic on 22.11.2023..
+//
+
+import Foundation
+import MessageUI
+
+@objcMembers
+public final class EmailSenderTool: NSObject, Tool {
+
+    // MARK: - Public properties -
+    
+    public var name: String { "Email Sender" }
+
+    // MARK: - Private properties -
+
+    private let getter: () -> (MailData)
+
+    private var alertTitle: String
+    private var alertMessage: String
+
+    // MARK: - Lifecycle -
+
+    public init(
+        getter: @escaping () -> (MailData),
+        alertTitle: String = "Email Not Available",
+        alertMessage: String = "Your device is not configured to send emails. Please set up an email account in Mail app or use another device."
```

Comment:
With this, a developer can put different title and message, which will not be the correct output if the email is not available (used for that use case below).

Reply:
I will explain it to the implementor in init comment